test/cbmc/proofs/findHeaderFieldParserCallback/findHeaderFieldParserCallback_harness.c

```diff
@@ -26,17 +26,17 @@
  */
```

Comment:
The makefile for this case needs a loop unwindset updated: https://d39z7xfr4hj3jh.cloudfront.net/final/f6b58be4-d13f-4665-9be6-c5c08cfb8476/artifacts/findHeaderFieldParserCallback/report/html/index.html

Reply:
I tested removing that unwindset and the test fails, so it is doing something despite what the warning indicates. I'll ask in the cbmc group.